e2e/flyout.e2e-spec.ts

```diff
@@ -95,4 +95,54 @@ describe('Flyout', () => {
     });
     element(by.css('.sky-flyout .sky-flyout-btn-close')).click();
   });
+
+  it('should match previous screenshot when their is responsive content (flyout: xs)', (done) => {
```

Comment:
`their` should be `there`. Maybe we could be more specific here and say, `...when the flyout contains responsive children` ?

Reply:
Done